controllers/podplacement/plugins_test.go

```diff
@@ -0,0 +1,34 @@
+package podplacement
+
+import (
+	"testing"
+
+	"github.com/openshift/multiarch-tuning-operator/apis/multiarch/common/plugins"
+)
+
+func TestBasePlugin_IsEnabled(t *testing.T) {
+	tests := []struct {
+		name     string
+		enabled  bool
+		expected bool
+	}{
+		{"Enabled Plugin", true, true},
+		{"Disabled Plugin", false, false},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			plugin := &plugins.BasePlugin{Enabled: tt.enabled}
+			if plugin.IsEnabled() != tt.expected {
```

Comment:
```suggestion
			if plugin.IsEnabled() != tt.enabled {
```

This unit test concers the correcteness of the `IsEnabled` function. Given the simplicity, it's ok not to duplicate the expected value and use `tt.enabled`.